fdbclient/FDBTypes.h

```diff
@@ -235,7 +235,7 @@ struct KeyRangeRef {
 		return *this;
 	}
 
-	int expectedSize() const { return begin.expectedSize() + end.expectedSize(); }
+	int expectedSize() const { return begin.expectedSize() + end.expectedSize() + sizeof(KeyRangeRef); }
```

Comment:
I don't think we should change the definition of `expectedSize`, but rather we should include this additional value in places where we need it.

Reply:
Done.

Reply:
Done.